hack/setup.sh

```diff
@@ -165,6 +165,50 @@ function captureArgoCDCreds {
   fi
 }
 
+function captureNotificationsEmailHostname {
+   if [ -z "$NOTIFICATIONS_EMAIL_HOSTNAME" ]; then
+    read -s -p "Enter the SMTP server hostname for notification emails (empty for disabling it): " value
+    echo ""
+    NOTIFICATIONS_EMAIL_HOSTNAME=$value
+  else
+    echo "SMTP server hostname for notification emails already set."
+  fi
+}
+
+function captureNotificationsEmailUsername {
+   if [ -z "$NOTIFICATIONS_EMAIL_USERNAME" ]; then
+    read -s -p "Enter the SMTP server username for notification emails (empty for disabling it): " value
```

Comment:
the -s will hide the input - better to not use it for username, and hostname

Reply:
agree. noticed that when I start testing the PR. only password should be hidden.